code/modules/mob/living/carbon/human/life/handle_shock.dm

```diff
@@ -2,28 +2,28 @@
 
 /mob/living/carbon/human/handle_shock()
 	. = ..()
-	if(status_flags & GODMODE || analgesic || (species && species.species_flags & NO_PAIN) || stat == DEAD)
+	if(status_flags & GODMODE || analgesic || (species && species.species_flags & NO_PAIN))
 		setShock_Stage(0)
 		return //Godmode or some other pain reducers. //Analgesic avoids all traumatic shock temporarily
 
 	switch(traumatic_shock)
 		if(200 to INFINITY)
-			setShock_Stage(max(shock_stage + 5, 150)) //Indescribable pain. At this point, you will immediately be knocked down, with shock stage set to 150.
+			adjustShock_Stage(5) //All of these adjust shock_stage based on traumatic_shock
```

Comment:
I don't like this change, this means you can get untreatable amount of pain

Reply:
It's capped to a max of 200 shock_stage. Additionally, oxycodone and three other chems set your shock_stage down to 150 if it's above that.